protected/humhub/modules/web/pwa/controllers/ManifestController.php

```diff
@@ -54,7 +54,7 @@ public function actionIndex()
     private function handlePwa()
     {
         $this->manifest['display'] = 'standalone';
-        $this->manifest['start_url'] = Url::home(true);
+        $this->manifest['start_url'] = Url::base(true);
```

Comment:
Maybe we should use `humhub\libs\SafeBaseUrl.php` here @luke-?

Reply:
@ArchBlood Thanks for review. SafeBaseUrl is only required, when we're sending E-Mails.